src/server/transaction/server/testing/server_test.go

```diff
@@ -237,22 +260,38 @@ func TestTransactions(suite *testing.T) {
 		ctx := pctx.TestContext(t)
 		env := realenv.NewRealEnv(ctx, t, dockertestenv.NewTestDBConfig(t))
 
+		project := testutil.UniqueString("prj_")
+		err := env.PachClient.CreateProject(project)
+		require.NoError(t, err)
+
 		txn, err := env.PachClient.StartTransaction()
 		require.NoError(t, err)
 
 		txnClient := env.PachClient.WithTransaction(txn)
 
-		err = txnClient.CreateProjectRepo(pfs.DefaultProjectName, "foo")
+		err = txnClient.CreateProjectRepo(project, "foo")
 		require.NoError(t, err)
 
-		_, err = txnClient.StartProjectCommit(pfs.DefaultProjectName, "foo", "master")
+		_, err = txnClient.StartProjectCommit(project, "foo", "master")
 		require.NoError(t, err)
-		err = txnClient.FinishProjectCommit(pfs.DefaultProjectName, "foo", "master", "")
+		err = txnClient.FinishProjectCommit(project, "foo", "master", "")
 		require.NoError(t, err)
 
-		_, err = txnClient.StartProjectCommit(pfs.DefaultProjectName, "foo", "master")
+		_, err = txnClient.StartProjectCommit(project, "foo", "master")
 		require.YesError(t, err)
 		require.Matches(t, "already has a commit in this transaction", err.Error())
+		//Delete and verify deletion occurs as well
```

Comment:
There should be a space between // and Delete.  Comments for humans have a space, comments for tooling (`//nolint:whatever`, `//go:build`, etc.) omit the space.